package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "service-runner",
-  "version": "2.2.5",
+  "version": "3.0.0",
```

Comment:
Perhaps better to bump the version in a separate commit as it will give us better visibility of the switch to version 3